todoman/cli.py

```diff
@@ -294,7 +296,13 @@ def move(ctx, list, ids):
 @click.option('--reverse/--no-reverse', default=True,
               help='Sort tasks in reverse order (see --sort). '
               'Defaults to true.')
-def list(ctx, lists, all, urgent, location, category, grep, sort, reverse):
+@click.option('--due', default=None, help='Only show tasks due in DUE hours',
+              type=int)
+# TODO: we might want a `porcelain` flag here to print this is a
```

Comment:
I was thinking something a bit simpler, like this (due date, percent, title). Basically inspired in `git status --porcelain`:

2017-01-01 0 Do stuff
2017-01-01 100 Do more stuff

---

JSON is nicer for apps to read, but a bit more complex for bash scripts. I guess it depends a bit on who we're targeting with this, but I'd much rather stick to the simplest-possible format.